### PR TITLE
added run test command suggestion for ts file

### PR DESCRIPTION
**Pull request by sadabnepal**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)
Fix of #6978 
## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Review by christian-bromann — CHANGES_REQUESTED** [review]
Rather than defining two almost identical messages I suggest to use `util.format`, e.g.:

```js
export const CONFIG_JS_HELPER_SUCCESS_MESSAGE = `
Configuration file was created successfully!
To run your tests, execute:
$ npx wdio run wdio.conf.%s
```

and then:

```js
console.log(util.format(CONFIG_TS_HELPER_SUCCESS_MESSAGE, answers.isUsingCompiler === COMPILER_OPTIONS.ts ? 'ts' : 'js'))
```

**Comment by christian-bromann** [comment]
Also please make sure the unit tests are passing. Thanks!

**Comment by sadabnepal** [comment]
> Also please make sure the unit tests are passing. Thanks!

@christian-bromann  I am new here and might need a bit of guidance on how to write unit test for this scenario.

File: packages/wdio-cli/src/commands/config.ts [hunk 1]
```diff
@@ -1,21 +1,21 @@
-import fs from 'fs'
-import path from 'path'
-import util from 'util'
-import inquirer from 'inquirer'
+import * as fs from 'fs'
+import * as path from 'path'
+import * as util from 'util'
+import * as inquirer from 'inquirer'
```
Comment (christian-bromann) [inline packages/wdio-cli/src/commands/config.ts:4]: Can we revert this? The way TypeScript is setup does not require such imports.

File: packages/wdio-cli/src/commands/config.ts [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+    CONFIG_HELPER_SUCCESS_MESSAGE,
 } from '../constants'
 import {
     addServiceDeps, convertPackageHashToObject, renderConfigurationFile,
     hasFile, generateTestFiles, getAnswers, getPathForFileGeneration,
     hasPackage
 } from '../utils'
 import { ConfigCommandArguments, ParsedAnswers } from '../types'
-import yargs from 'yargs'
+import * as yargs from 'yargs'
```
Comment (christian-bromann) [inline packages/wdio-cli/src/commands/config.ts:18]: same

File: packages/wdio-cli/src/commands/config.ts [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+import * as path from 'path'
+import * as util from 'util'
+import * as inquirer from 'inquirer'
 import yarnInstall from 'yarn-install'
 
 import {
     CONFIG_HELPER_INTRO, CLI_EPILOGUE, COMPILER_OPTIONS,
     TS_COMPILER_INSTRUCTIONS, SUPPORTED_PACKAGES,
-    CONFIG_HELPER_SUCCESS_MESSAGE
+    CONFIG_HELPER_SUCCESS_MESSAGE,
```
Comment (christian-bromann) [inline packages/wdio-cli/src/commands/config.ts:10]:
```suggestion
    CONFIG_HELPER_SUCCESS_MESSAGE
```

**Review by christian-bromann — CHANGES_REQUESTED** [review]
The code looks good, now updating the unit test is important.

File: packages/wdio-cli/src/commands/config.ts (hunk 2 above)
Comment (sadabnepal) [reply packages/wdio-cli/src/commands/config.ts]: Reverted

File: packages/wdio-cli/src/commands/config.ts (hunk 3 above)
Comment (sadabnepal) [reply packages/wdio-cli/src/commands/config.ts]: removed comma

**Comment by sadabnepal** [comment]
Closing this PR, I will create new one as this is failing on sample run

**Comment by christian-bromann** [comment]
@sadabnepal can you point me to the new PR?

**Pull request closed without merging**